components/scream/src/physics/p3/atmosphere_microphysics.cpp

```diff
@@ -179,21 +179,21 @@ void P3Microphysics::init_buffers(const ATMBufferManager &buffer_manager)
 }
 
 // =========================================================================================
-void P3Microphysics::initialize_impl (const util::TimeStamp& t0)
+void P3Microphysics::initialize_impl (const util::TimeStamp& /* t0 */)
 {
   // Set property checks for fields in this process
-  auto positivity_check = std::make_shared<FieldPositivityCheck<Real> >();
-  m_fields_out["qv"].add_property_check(positivity_check);
-  m_fields_out["qc"].add_property_check(positivity_check);
-  m_fields_out["qr"].add_property_check(positivity_check);
-  m_fields_out["qi"].add_property_check(positivity_check);
-  m_fields_out["qm"].add_property_check(positivity_check);
-  m_fields_out["nc"].add_property_check(positivity_check);
-  m_fields_out["nr"].add_property_check(positivity_check);
-  m_fields_out["ni"].add_property_check(positivity_check);
-  m_fields_out["bm"].add_property_check(positivity_check);
+  // auto positivity_check = std::make_shared<FieldPositivityCheck<Real> >();
+  // get_field_out("qv").add_property_check(positivity_check);
```

Comment:
Did we mean to keep these commented out?

Reply:
I guess I can remove them. I wasn't sure what to do with the FPC's in Homme, so I commented for now. You are working on fixing these checks, right? Does it bother you to keep them in master just for a couple more days, till you get your PR ready?